applications/convection_diffusion_application/custom_elements/laplacian_element.cpp

```diff
@@ -76,28 +75,41 @@ void LaplacianElement::CalculateLocalSystem(MatrixType& rLeftHandSideMatrix, Vec
     //reading integration points and local gradients
     const GeometryType::IntegrationPointsArrayType& integration_points = GetGeometry().IntegrationPoints();
     const GeometryType::ShapeFunctionsGradientsType& DN_De = GetGeometry().ShapeFunctionsLocalGradients();
-    
+    const Matrix& N_gausspoint = GetGeometry().ShapeFunctionsValues();
```

Comment:
mmm
this is the Ns on all the gauss points. use the plural in the name or a better name. asideo fo this looks ok to me, thx